docker/services/cron/crontab

```diff
@@ -36,3 +36,6 @@ MAILTO=""
 0 4 1,15 * * root flock -x -n /var/lock/lb-dumps.lock /code/listenbrainz/admin/create-dumps.sh full >> /logs/dumps.log 2>&1
 ## Around 24 hours later, trigger a full import into the spark cluster, and this time wait for the lock, in case the dump hasn't finished.
 0 4 2,16 * * root flock -x /var/lock/lb-dumps.lock /usr/local/bin/python /code/listenbrainz/manage.py spark request_import_full >> /logs/dumps.log 2>&1
+
+# run weekly cron job to remove expired do not recommends
```

Comment:
Normally I would say that weekly is not precise enough, but in this case who is ever going to know? :laughing:

Reply:
The retrieve query already checks until field against current datetime so it will always return precise entries. This job is only there to remove expired entries periodically.